app/static/js/studentMinorPage.js

```diff
@@ -1,28 +1,56 @@
 $(document).ready(function(){
-    $("#expressInterest").on("click", function() {
+    
+
+    $("#summerExperienceSave").click(function(){
+      let data = {'summerExperience': $("#summerExperience").val(),
+                  'selectedSummerTerm': $("#summerExperienceTerm").find(":selected").text()}
       let username = $("#username").val()
-      let data = {"username":username}
       $.ajax({
-          url: "/cceMinor/"+username+"/indicateInterest",
-          type: "POST",
-          data: data,
-          success: function(s) {
-            console.log(s)
+        type: "POST", 
+        url: "/cceMinor/"+username+"/addSummerExperience",
+        data: data,
+        success: function(s){
+          msgToast("Success!", "Summer Experience successfully added!")
+        },
+        error: function(error){
+          msgToast("Error!", "Error adding summer experience!")
+        }
+      })
 
-            location.reload()
-          },
-          error: function(request, status, error) {
-            console.log(error)
-            msgFlash("Error saving changes!", "danger")
-          }
-      });
-    })
+    });
+    $("#removeSummerExperience").click(function(){
+      let username = $("#username").val()
+      $.ajax({
+        type: "POST", 
+        url: "/cceMinor/"+username+"/deleteSummerExperience",
+        success: function(s){
+          $("#summerExperience").val("")
+          msgToast("Success!", "Summer Experience successfully deleted!")
+        },
+        error: function(error){
+          msgToast("Error!", "Error deleting summer experience!")
```

Comment:
Like previous comment, "Summer Experience" should be capitalized in both instances.